base/error.jl

```diff
@@ -23,7 +23,8 @@ error(s...) = throw(Main.Base.ErrorException(Main.Base.string(s...)))
 
 rethrow() = ccall(:jl_rethrow, Void, ())::Bottom
 rethrow(e) = ccall(:jl_rethrow_other, Void, (Any,), e)::Bottom
-backtrace() = ccall(:jl_backtrace_from_here, Array{Ptr{Void},1}, ())
+backtrace() = ccall(:jl_backtrace_from_here, Array{Ptr{Void},1}, (Int32,), false)
+backtrace2() = ccall(:jl_backtrace_from_here, Ref{SimpleVector}, (Int32,), true)
```

Comment:
it's a one-liner ccall, if `backtrace2` is a temporary wrapper than it can just be called directly via the `ccall` for now I think